lib/matplotlib/dviread.py

```diff
@@ -678,57 +679,66 @@ def _read(self):
         Read one page from the file. Return True if successful,
         False if there were no more pages.
         """
-        packet_char, packet_ends = None, None
-        packet_len, packet_width = None, None
+        packet_char = packet_ends = None
+        packet_len = packet_width = None
         while True:
             byte = self.file.read(1)[0]
             # If we are in a packet, execute the dvi instructions
             if self.state is _dvistate.inpage:
                 byte_at = self.file.tell()-1
                 if byte_at == packet_ends:
                     self._finalize_packet(packet_char, packet_width)
-                    packet_len, packet_char, packet_width = None, None, None
+                    packet_len = packet_char = packet_width = None
                     # fall through to out-of-packet code
                 elif byte_at > packet_ends:
                     raise ValueError("Packet length mismatch in vf file")
                 else:
                     if byte in (139, 140) or byte >= 243:
-                        raise ValueError(
-                            "Inappropriate opcode %d in vf file" % byte)
+                        raise ValueError(f"Inappropriate opcode {byte} in vf file")
                     Dvi._dtable[byte](self, byte)
                     continue
 
             # We are outside a packet
             if byte < 242:          # a short packet (length given by byte)
                 packet_len = byte
-                packet_char, packet_width = self._arg(1), self._arg(3)
+                packet_char = self._arg(1)
+                packet_width = self._arg(3)
                 packet_ends = self._init_packet(byte)
                 self.state = _dvistate.inpage
             elif byte == 242:       # a long packet
-                packet_len, packet_char, packet_width = \
-                            [self._arg(x) for x in (4, 4, 4)]
+                packet_len = self._arg(4)
+                packet_char = self._arg(4)
+                packet_width = self._arg(4)
```

Comment:
```suggestion
                packet_len = packet_char = packet_width = self._arg(4)
```
?

Similar below. (Assuming that calling `_arg` doesn't have some sort of state.)

Reply:
Yes, it reads more data from `self.file`. Maybe I should rename it, too...

Reply:
I renamed it to `_read_arg`.

Reply:
I think that was a good move! Although a bit sloppy on my side, not checking what it did, the changed name makes it more clear that it is not just getting a static item.